### PR TITLE
#383 Add aliases to actor

### LINKED ISSUE #383: Add `aliases` field to `actor` model
The Threat Intel team is working on adding a Mitre Groups importer to Hydrant. The source data includes an `aliases` field, but there is no destination in the CTIM `actor` model. The STIX Threat Actor model does include an `aliases` field.

CTIM Actor model: https://github.com/threatgrid/ctim/blob/master/doc/structures/actor.md
STIX Threat Actor model: https://docs.oasis-open.org/cti/stix/v2.1/csprd01/stix-v2.1-csprd01.html#_Toc16070663

### PR DESCRIPTION
**Pull request by samwagg**
closes #383

### CODE REVIEW

File: src/ctim/examples/actors.cljc [hunk 1]
```diff
@@ -38,7 +38,8 @@
    :planning_and_operational_support "foo"
    :valid_time {:start_time #inst "2016-02-11T00:40:48.212-00:00"
                 :end_time #inst "2016-07-11T00:40:48.212-00:00"}
-   :tlp "green"})
+   :tlp "green"
+   :aliases ["alias 1" "alias 2"]})
```
Comment (msprunck) [inline src/ctim/examples/actors.cljc:42]: You also need to change the schema ;)

Comment (samwagg) [reply src/ctim/examples/actors.cljc:42]: oops! Tried to fix and committed garbage. Hold on...

File: src/ctim/schemas/actor.cljc [hunk 2]
```diff
@@ -31,7 +31,8 @@
    (f/entry :sophistication v/Sophistication)
    (f/entry :intended_effect v/IntendedEffect)
    (f/entry :planning_and_operational_support c/LongString)
-   (f/entry :confidence v/HighMedLow))
+   (f/entry :confidence v/HighMedLow)
+   (f/entry :aliases [c/ShortString]))
```
Comment (msprunck) [inline src/ctim/schemas/actor.cljc:35]:
```suggestion
   (f/entry :aliases [c/ShortString]
                 description (str "A list of other names that this Threat Actor is "
                                            "believed to use.")))
```

**Review by msprunck — APPROVED** [review]
LGTM

File: src/ctim/schemas/actor.cljc (hunk 2 above)
Comment (samwagg) [reply src/ctim/schemas/actor.cljc]: Updated

**Comment by samwagg** [comment]
Can this be merged?

**Comment by samwagg** [comment]
> > Can this be merged?
> 
> yes!

Do we not need a version bump on this one?

**Comment by msprunck** [comment]
> > > Can this be merged?
> > 
> > 
> > yes!
> 
> Do we not need a version bump on this one?

No it will be included in the same release as https://github.com/threatgrid/ctim/pull/384

**Comment by samwagg** [comment]
Okay, sounds good! I don't seem to have permission to do the merge, just fyi.

**Pull request merged**